crates/core/c8y_api/src/smartrest/message_utils.rs

```diff
@@ -0,0 +1,24 @@
+/// Extract the Device ID from the SmartREST payload.
```

Comment:
You could even call this module `message.rs` just like `topic.rs`.

Reply:
This a good idea to add this module. It would be good to move there other smartrest related function as `get_smartrest_template_id` currently in `tedge_mapper/src/c8y/converter.rs`.


As @albinsuresh , I prefer the name `message` for that module. [Minor]

Reply:
Commited.